### PR TITLE
Add config for Quest import

### PR DESCRIPTION
**Pull request by BillWagner**
This PR adds the configuration for the Quest import GitHub action.

Before merging this PR, follow the instructions [here](https://github.com/dotnet/docs-tools/tree/main/actions/sequester#installation-and-use) to create the API KEY secrets. Then, once this is installed, it will start updating work items.

Note: To import items in bulk, when those have already been labeled, you can run the `quest-bulk` action via the GitHub UI to import all issues already labeled.

### CODE REVIEW

File: .github/workflows/quest-bulk.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        id: bulk-sequester
+        uses: dotnet/docs-tools/actions/sequester@main
+        env:
+          ImportOptions__ApiKeys__GitHubToken: ${{ secrets.GITHUB_TOKEN }}
+          ImportOptions__ApiKeys__OSPOKey: ${{ secrets.OSPO_KEY }}
+          ImportOptions__ApiKeys__QuestKey: ${{ secrets.QUEST_KEY }}
+        with:
+          org: ${{ github.repository_owner }}
+          repo: ${{ github.repository }}
+
```
Comment (IEvangelist) [inline .github/workflows/quest-bulk.yml:34]:
```suggestion
          repo: ${{ github.repository }}
          issue: '-1'
```

**Comment by Rick-Anderson** [comment]
@BillWagner @IEvangelist I created the two API KEY/secret pairs, where do I register the secret?

**Comment by IEvangelist** [comment]
> @BillWagner @IEvangelist I created the two API KEY/secret pairs, where do I register the secrets?

Hey @Rick-Anderson, see https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository

**Comment by Rick-Anderson** [comment]
@BillWagner @IEvangelist  can we merge and test?

**Comment by IEvangelist** [comment]
> @BillWagner @IEvangelist can we merge and test?

Give us 10 mins

**Comment by BillWagner** [comment]
> 

Merge anytime.

**Pull request merged**